rust/rope/src/tree.rs

```diff
@@ -566,33 +579,74 @@ impl<'a, N: NodeInfo> Cursor<'a, N> {
         }
     }
 
+    // Moves the cursor to the next discontinuity, or to the end
```

Comment:
`///` because this is fine as doc string.

I think I've been using "boundary" instead of "discontinuity," but not sure exactly which is better. Should be consistent though.